pywren/pywren_ibm_cloud/storage/storage.py

```diff
@@ -124,7 +125,17 @@ def get_call_output(self, executor_id, callgroup_id, call_id):
             return self.backend_handler.get_object(self.storage_bucket, output_key)
         except StorageNoSuchKeyError:
             return None
-    
+
+    def get_calls_ids(self, executor_id, callgroup_id):
```

Comment:
@omerb01 I think the name of this method is a bit confusing. if you move the logic from` get_group_calls_ids()` here, then you can rename this method to `get_group_calls_ids()` or` get_callgroup_calls_ids()`

Reply:
thanks, will change its name to `get_callgroup_calls_ids()`